app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/model/DataSetInterface.kt

```diff
@@ -0,0 +1,9 @@
+package com.github.HumanLearning2021.HumanLearningApp.model
+
+
+/**
+ * An interface representing the part of the model interacting with data sets
+ */
+interface DataSetInterface {
```

Comment:
Maybe a question of taste, but I wouldn't have put a capital 'S' in Dataset (https://en.wikipedia.org/wiki/Data_set)
(but it's not a big problem)

Reply:
done